build_projects/shared-build-targets-utils/Utils/HostVersion.cs

```diff
@@ -133,7 +133,7 @@ public VerInfo GetLockedHostFXRPlatformInstallerVersion()
 
             if (fExplicitHostFXRMSIVersion && CurrentPlatform.Current == BuildPlatform.Windows)
             {
-                version = new VerInfo(1, 1, 2, "", "", "", CommitCountString);
```

Comment:
Should we also flip `fExplicitHostFXRMSIVersion` to `false`?  that way the MSI version stays in step with `LockedHostFxrVersion`.

Reply:
The explicit version was set to prevent HostFXR from getting new versions when there is minimal churn in HostFXR. 
Assuming there is new change in hotsfxr now, I think setting the value to 1.1.9 seems right. 

We don't want to revert to LockedHostFxrVersion as IsLocked is not yet locked. It has most probably not set to true ever.

Reply:
So we should keep `fExplicitHostFXRMSIVersion=true` ?

Reply:
Right.